jenkins/databricks/install_deps.py

```diff
@@ -42,6 +42,9 @@ def define_deps(spark_version, scala_version):
     elif spark_version.startswith('3.4'):
         spark_prefix = '----ws_3_4'
         mvn_prefix = '--mvn'
+    elif spark_version.startswith('3.5'):
+        spark_prefix = '----ws_3_5'
+        mvn_prefix = '--mvn'
```

Comment:
Do we want an else here that fails indicating an unsupported Databricks version was detected?  As-is, it looks like it will just silently fall through and give a less obvious error.